assets/js/blocks/store-notices/edit.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import { useBlockProps } from '@wordpress/block-editor';
+import { Disabled } from '@wordpress/components';
+import type { BlockEditProps } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import Block from './block';
+
+export interface Attributes {
+	className?: string;
+}
+
+const Edit = ( { attributes }: BlockEditProps< Attributes > ) => {
+	const { className } = attributes;
+	const blockProps = useBlockProps( {
+		className: classNames( 'wc-block-store-notices', className ),
+	} );
+
+	return (
+		<>
+			<div { ...blockProps }>
+				<Disabled>
+					<Block attributes={ attributes } />
```

Comment:
This line causes a TS error, `StoreNotices` doesn't accept any props. I had similar case in Product Results Count, so you can either re-use [this commit](https://github.com/woocommerce/woocommerce-blocks/pull/8078/commits/de6febf071653c72c5bec16f0a0ab0834bd233b5) or avoid passing attributes down to the block.

Reply:
Fixed!